process/support_team.md

```diff
@@ -0,0 +1,46 @@
+# The Made Tech Support Team
+
+## High level overview
+
+Our Support Team is best thought of as the 'on-demand' equivalent to our AaaS offering. They undertake smaller tactical pieces of work which normally would necessitate the spinning up of a dedicated Sprint team. The length of work undertaken varies from less than an hour, up to about a month. Beyond that our AaaS offering becomes more appropriate.
+
+The support team doesn't have a fixed roster of staff and instead consists of a number of developers based availibility of staff and customer demand for support work. 
+
+The Support Team is run by Nick Wood, who oversees day to day operations, and manages relationships with our on-going support customers.  
+
+## Support issue lifecycle
+
+All support issues begin life as an email into support@madetech.com. These are forwarded automatically into our GrooveHQ inbox to be sorted by the Triage Team. We aim to resolve all new issues within 48 hours with one of the following resolutions:
+* Fix issue if it can be done so in < 2 hours
+* Fix issue if it is causing significant business impact to customer
+* If it is a larger piece of work, and/or non-urgent then it can be moved to the customer's backlog in Trello.
+
+Each Customer has their own Trello board for prioritising work. Issues are grouped differently for each customer but each board has the following 3 columns:
+1. 'New Issues' this is anything that's been backlogged from a Groove issue.
+2. 'Up next' issues will be automatically copied to our Support Epics board
+3. 'Recently Completed' issues are things which have been completed in the last week or so.
+
+During a weekly (bi-weekly for some customers) call with each customer Nick and the custmer will agree to a tentative commitment of work to be completed the following week by the support team. These issues will be moved to the 'Up Next' column, and copied to the Support Epics board.
+
+Every Monday everyone working on support that week will have a planning session to divide up the work for our customers across the team. Following this session Nick will follow up with each costomer to confirm what's going to be done that week.
```

Comment:
True, but in the new world these 3 hour jobs, could hopefully be tackled by the triage team, leaving the rest of support to pick up the meaty stuff